src/components/AddNewInventory/AddNewInventory.jsx

```diff
@@ -1,11 +1,214 @@
+    }
+  };
+
+  return (
+    <>
+      <section className="new-inventory">
+        <InventoryFormHeader>Add New Inventory Item</InventoryFormHeader>
+        <form className="new-inventory__form" onSubmit={handleSubmit}>
+          <div className="new-inventory__item-details">
+            <h2>Item Details</h2>
+            <label>Item Name</label>
+            <input
+              type="text"
+              name="itemName"
+              placeholder="Item Name"
+              className={`new-inventory__name-input ${
+                error && !itemName ? "inventory-form__input--error" : ""
+              } `}
+              id="itemNameInput"
+              value={itemName}
+              onChange={(e) => setItemName(e.target.value)}
+            />
+            <FormError errorState={error} field={itemName}>
+              Item is required
+            </FormError>
+            <label>Description</label>
+            <input
+              type="text"
+              name="description"
+              placeholder="Please enter a brief item description..."
+              className={`new-inventory__description-input ${
+                error && !description ? "inventory-form__input--error" : ""
+              }`}
+              id="descriptionInput"
+              value={description}
+              onChange={(e) => setDescription(e.target.value)}
+            />
+            <FormError errorState={error} field={description}>
+              Description is required
+            </FormError>{" "}
+            <label>Category</label>
+            <div className="new-inventory__dropdown">
+              <select
+                value={category}
+                onChange={(e) => setCategory(e.target.value)}
+                className={`inventory-form__input inventory-form__input--select
+                  ${error && !category ? "inventory-form__input--error" : ""}`}
+              >
+                <option value="" disabled>
+                  Please select
+                </option>
+                <option value="electronics">Electronics</option>
+                <option value="gear">Gear</option>
+                <option value="apparel">Apparel</option>
+                <option value="accessories">Accessories</option>
+                <option value="health">Health</option>
+              </select>
+              <FormError errorState={error} field={category}>
+                Category is required
+              </FormError>
+            </div>
+          </div>
+
+          <div className="new-inventory__item-avail">
+            <h2>Item Availability</h2>
+            <label>Status</label>
+            <div className="new-inventory__radio-btns">
+              <input
+                type="radio"
+                name="status"
+                value="inStock"
+                className="new-inventory__radio-btn"
+                id="inStock"
+                onChange={() => setStatus("inStock")}
```

Comment:
Correct the instock string as it is not consistent with the database string and hence its is not displayed correctly